onnxruntime/core/mlas/lib/activate.cpp

```diff
@@ -498,16 +498,19 @@ Return Value:
             break;
         }
 
-        case MlasClipActivation:
-        {
+        case MlasClipActivation: {
```

Comment:
>{ [](http://example.com/codeflow?start=33&length=1)

nit: mlas's format is different with ORT's other part. Please follow the existing format for now.

Reply:
Fixed.